### ISSUE TITLE
Credentials reset after reboot

### ISSUE THREAD

**Issue by RaptaG** [flathub/io.vikunja.Vikunja#7]
Possibly a missing permission

**Comment by kolaente** [comment]
Is this only happening with the flathub version of Vikunja?

**Comment by RaptaG** [comment]
I have not tested the native client yet, so I don't know

**Comment by kolaente** [comment]
Maybe this is related? https://github.com/go-vikunja/desktop/issues/12#issuecomment-1863317018

**Comment by RaptaG** [comment]
Will test again with the new version to if the issue persists

**Comment by RaptaG** [comment]
Getting an

```
Could not find or use Vikunja installation at "app.vikunja.cloud". Please check if the url has the correct format and you can reach it when accessing it directly and try again.
```
error when trying to log in @kolaente

**Comment by kolaente** [comment]
@RaptaG Does it work if you enter `https://app.vikunja.cloud` or `https://app.vikunja.cloud/api/v1`?

**Comment by RaptaG** [comment]
> @RaptaG Does it work if you enter `https://app.vikunja.cloud` or `https://app.vikunja.cloud/api/v1`?

The second one worked

**Comment by RaptaG** [comment]
After logging in, closing the app and then opening it again the following error shows up;

```
{"error":"invalid_client","error_description":"Client authentication failed","message":"Client authentication failed"}
```

**Comment by kolaente** [comment]
Is that the same case when you use Vikunja in the browser?

**Comment by RaptaG** [comment]
> Is that the same case when you use Vikunja in the browser?

No, but from what I can tell the credentials reset on my browser as well

**Comment by kolaente** [comment]
Does that happen after you click on the "Login with Vikunja Cloud" link?

**Comment by RaptaG** [comment]
> Does that happen after you click on the "Login with Vikunja Cloud" link?

Yes
